meilisearch-lib/src/error.rs

```diff
@@ -41,6 +41,7 @@ impl ErrorCode for MilliError<'_> {
                     UserError::CriterionError(_) => Code::InvalidRankingRule,
                     UserError::InvalidGeoField { .. } => Code::InvalidGeoField,
                     UserError::SortError(_) => Code::Sort,
+                    UserError::InvalidMinTypoWordLenSetting(_, _) => unreachable!(),
```

Comment:
Not sure but, it should be better to return at least an internal error, no? 🤔

Reply:
this error cannot be reached now, but will in two PR :) (#2258)

Reply:
Ah ok! Thanks!